astropy/units/core.py

```diff
@@ -1930,7 +1930,7 @@ class Unit(NamedUnit, metaclass=_UnitMetaClass):
 
```

Comment:
Sorry for the hassle, but could you also change the line above (now `From None`) to `From no arguments:`

Maybe also rebase so the tests pass again...

Reply:
Should be good now